source/slang/slang-ast-builder.cpp

```diff
@@ -545,10 +545,12 @@ PtrTypeBase* ASTBuilder::getPtrType(
     AddressSpace addrSpace,
     char const* ptrTypeName)
 {
+    Type* typeOfAccessQualifier = getSpecializedBuiltinType({}, "AccessQualifier");
```

Comment:
Maybe good to cache these types inside ASTBuilder so we don't do repetitive lookups every time it is needed.